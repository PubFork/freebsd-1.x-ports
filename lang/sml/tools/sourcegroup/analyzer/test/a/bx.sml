structure S = struct
  open F
  val b = B.x
  end
