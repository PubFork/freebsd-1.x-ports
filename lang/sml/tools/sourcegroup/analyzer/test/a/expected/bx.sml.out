source sml "a/bx.sml"
  import structure B F 
  export structure S ;

